x-pack/plugins/index_lifecycle_management/common/types/index.ts

```diff
@@ -9,8 +9,13 @@ export * from './api';
 export * from './policies';
 
 /**
- * These roles reflect how nodes are stratified into different data tiers. The "data" role
- * is a catch-all that can be used to store data in any phase.
+ * These roles reflect how nodes are stratified into different data tiers.
+ *
  */
 export type NodeDataRole = 'data_hot' | 'data_warm' | 'data_cold';
-export type NodeDataRoleWithCatchAll = 'data' | NodeDataRole;
+
+/**
+ * The "data" role can store data on any tier and the "data_content" role can store
+ * all data the ES stack uses for feature functionality like security related indices.
+ */
+export type NodeDataRoleWithCatchAll = 'data' | 'data_content' | NodeDataRole;
```

Comment:
Nit: not sure the name makes sense now. Perhaps it would make more sense to call this `AnyNodeRole` and rename `NodeDataRole` to `DataTierNodeRole`?